[2021-02-14 16:06:01,276] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: NDBC_51001.clean_and_load 2021-02-15T02:05:54.220515+00:00 [queued]>
[2021-02-14 16:06:01,281] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: NDBC_51001.clean_and_load 2021-02-15T02:05:54.220515+00:00 [queued]>
[2021-02-14 16:06:01,281] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-02-14 16:06:01,281] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-02-14 16:06:01,281] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-02-14 16:06:01,286] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): clean_and_load> on 2021-02-15T02:05:54.220515+00:00
[2021-02-14 16:06:01,289] {standard_task_runner.py:52} INFO - Started process 94964 to run task
[2021-02-14 16:06:01,295] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'NDBC_51001', 'clean_and_load', '2021-02-15T02:05:54.220515+00:00', '--job-id', '36', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/ndbc_dag.py', '--cfg-path', '/var/folders/y8/ykx6qdr96dldyw_z287s5l580000gn/T/tmp0vsnrmes', '--error-file', '/var/folders/y8/ykx6qdr96dldyw_z287s5l580000gn/T/tmpe0yifbod']
[2021-02-14 16:06:01,297] {standard_task_runner.py:77} INFO - Job 36: Subtask clean_and_load
[2021-02-14 16:06:01,325] {logging_mixin.py:104} INFO - Running <TaskInstance: NDBC_51001.clean_and_load 2021-02-15T02:05:54.220515+00:00 [running]> on host chriss-mbp.lan1
[2021-02-14 16:06:01,344] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=holloway.christopher.t@gmail.com
AIRFLOW_CTX_DAG_OWNER=chris
AIRFLOW_CTX_DAG_ID=NDBC_51001
AIRFLOW_CTX_TASK_ID=clean_and_load
AIRFLOW_CTX_EXECUTION_DATE=2021-02-15T02:05:54.220515+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-02-15T02:05:54.220515+00:00
[2021-02-14 16:06:01,684] {base.py:65} INFO - Using connection to: id: NDBC_51001. Host: localhost, Port: 5400, Schema: NDBC_51001DB, Login: chrisholloway, Password: None, extra: None
[2021-02-14 16:06:01,689] {dbapi.py:180} INFO - Running statement: INSERT INTO Detailed_Wave_Data_Table (significant_wave_height, swell_height, swell_period,
	 wind_wave_height, wind_wave_period, average_period) VALUES (%s, %s, %s, %s, %s, %s);, parameters: (2.9, 2.9, 13.8, 0.6, 4.8, 9.5)
[2021-02-14 16:06:01,691] {dbapi.py:186} INFO - Rows affected: 1
[2021-02-14 16:06:01,694] {python.py:118} INFO - Done. Returned value was: None
[2021-02-14 16:06:01,699] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=NDBC_51001, task_id=clean_and_load, execution_date=20210215T020554, start_date=20210215T020601, end_date=20210215T020601
[2021-02-14 16:06:01,710] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-02-14 16:06:01,735] {local_task_job.py:146} INFO - Task exited with return code 0
